src/App.js

```diff
@@ -6,17 +6,24 @@ import Header from "./components/Header";
 import TopicArticles from "./components/TopicArticles";
 import Articles from "./pages/Articles";
 import Topics from "./pages/Topics";
+import ErrorScreen from "./components/ErrorScreen";
+import Home from "./pages/Home";
+import Users from "./pages/Users";
 
 function App() {
   return (
-    <UserContext.Provider value='jessjelly'>
+    <UserContext.Provider value="jessjelly">
       <div className="App">
         <Header />
         <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/home" element={<Home />} />
+          <Route path="/users" element={<Users />} />
           <Route path="/articles" element={<Articles />} />
           <Route path="/topics" element={<Topics />} />
           <Route path="/topic/:topic/articles" element={<TopicArticles />} />
           <Route path="/articles/:article_id" element={<ArticleView />} />
+          <Route path="*" element={<ErrorScreen type={"page"} />} />
```

Comment:
This doesn't _quite_ work! If you want it to work for topics, try this line instead:

` <Route path="/topics/*" element={<ErrorScreen type={"topics"} />} />`

You can use the `/*` endpoint after anything, which will bring up the error screen. Really good idea though!